node/service/lib/device.d.ts

```diff
@@ -24,9 +24,14 @@ declare namespace Device {
         secondaryKey: string;
     }
 
+    interface _X509Thumbprint {
+        primaryThumbprint: string;
+        secondaryThumbprint: string;
+    }
+
     interface Authentication {
-        SymmetricKey?: _SymmetricKey;
-        x509Thumbprint?: string;
+        symmetricKey?: _SymmetricKey;
```

Comment:
You're right. I don't know why I thought this was fixed in the JS already. Will undo this and update the PR. Thanks!